packages/itwinui-react/src/core/DropdownMenu/DropdownMenu.tsx

```diff
@@ -26,12 +37,16 @@ export type DropdownMenuProps = {
    * Child element to wrap dropdown with.
    */
   children: React.ReactNode;
-} & Omit<PopoverProps, 'content'> &
-  Omit<CommonProps, 'title'>;
+} & Pick<
+  React.ComponentProps<typeof Popover>,
```

Comment:
Is there a reason why you are using Popover for types here and `Parameters<typeof usePopover>[0]` in Select?

Reply:
hmm, maybe this one could also use `usePopover`.

i'm also wondering if we should decide on which props to expose and create a helper for that